platform/kubernetes/kubernetes.go

```diff
@@ -79,43 +76,35 @@ func isAddon(obj namespacedLabeled) bool {
 
 // --- /add ons
 
+type Applier interface {
+	Delete(logger log.Logger, def []byte) error
+	Create(logger log.Logger, def []byte) error
+	Apply(logger log.Logger, def []byte) error
```

Comment:
So the reason for distinguishing between Create and Apply comes down to: is it useful and possible to _just have upsert_?